samples/cellular/nrf_cloud_coap_cell_location/README.rst

```diff
@@ -0,0 +1,124 @@
+In the multi-cell mode, the sample requests for neighbor cell measurement using the :ref:`lte_lc_readme` library.
+If neighbor cell data is measured, the sample sends a multi-cell location request to nRF Cloud.
+Otherwise, the request is single-cell.
+In either mode, the sample sends a new location request if a change in cell ID is detected.
+
+See the `nRF Cloud Location Services documentation`_ for additional information.
+
+Limitations
+***********
+
+The sample requires the network carrier to provide date and time to the modem.
+Without a valid date and time, the modem cannot generate JWTs with an expiration time.
+
+User interface
+**************
+
+Pressing **Button 1** toggles between the single-cell and multi-cell mode.
+
+Set the :ref:`CONFIG_COAP_CELL_CHANGE_CONFIG <CONFIG_COAP_CELL_CHANGE_CONFIG>` Kconfig option value to try all combinations of the :c:struct:`nrf_cloud_location_config` structure values ``hi_conf`` and ``fallback``.
+
+.. _nrf_cloud_coap_cell_location_onboarding:
+
+Onboarding your device to nRF Cloud
+***********************************
+
+You must onboard your device to nRF Cloud for this sample to function.
+You only need to do this once for each device.
+To onboard your device, install `nRF Cloud Utils`_ and follow the instructions in the README.
+
+Configuration
+*************
+
+|config|
+
+
+Configuration options
+=====================
+
+Check and configure the following Kconfig options for the sample:
+
+.. _CONFIG_COAP_CELL_CHANGE_CONFIG:
+
+CONFIG_COAP_CELL_CHANGE_CONFIG - Enable changing request configuration
+   Set this to use the next combination of ``hi_conf`` and ``fallback`` flags after performing both single- and multi-cell location requests.
+
+.. _CONFIG_COAP_CELL_DEFAULT_DOREPLY_VAL:
+
+CONFIG_COAP_CELL_DEFAULT_DOREPLY_VAL - Enable return of location from cloud
+   If enabled, request the cloud to return the location information.
+
+.. _CONFIG_COAP_CELL_DEFAULT_FALLBACK_VAL:
+
+CONFIG_COAP_CELL_DEFAULT_FALLBACK_VAL - Enable fallback to coarse location
+   If enabled and the location of the cell tower or Wi-Fi® access points cannot be found, return area-level location based on the cellular tracking area code.
+   Otherwise an error will be returned indicating location is not known.
+
+.. _CONFIG_COAP_CELL_DEFAULT_HICONF_VAL:
+
+CONFIG_COAP_CELL_DEFAULT_HICONF_VAL - Enable high-confidence result
+   Enable a 95% confidence interval for the location, instead of the default 68%.
+
+.. _CONFIG_COAP_CELL_SEND_DEVICE_STATUS:
+
+CONFIG_COAP_CELL_SEND_DEVICE_STATUS - Send device status
+   Send device status to nRF Cloud on initial connection.
+
+.. include:: /libraries/modem/nrf_modem_lib/nrf_modem_lib_trace.rst
+   :start-after: modem_lib_sending_traces_UART_start
+   :end-before: modem_lib_sending_traces_UART_end
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/cellular/nrf_cloud_coap_cell_location`
+
+.. include:: /includes/build_and_run_ns.txt
+
```

Comment:
Is there a way of testing the sample? Testing is a mandatory section in a sample doc: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/templates/sample_README.html